### ISSUE TITLE
Coach view: update `<ReportsPracticeQuizQuestionPage>` ("difficult questions" report)

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#8517]
### User stories for this issue:

1. As a coach, when I select an attempt, I want to see more detailed information when I click into a difficult question so that I can see information for each learner for how they did this time compared to other times.

**From Coach > Reports > Lessons (click on lesson title) > Report > Channel quiz (click on channel quiz title) > Difficult questions > Question (click on question title)**

|Frame|Explanation|
|--|--|
|![Screen Shot 2021-10-21 at 12 02 53 PM](https://user-images.githubusercontent.com/13563002/138356681-360ef155-0d0e-4fca-9e9d-f2a9bb98766d.png)|Arrows show new or changed UI|

### Notes and changes to current designs:
- [Figma for viewing an independent practice quiz learner report](https://www.figma.com/file/TfaSTT9gs6fPKhGdNUjffU/Channel-quizzes?node-id=999%3A7526)
- All strings required for this component should already be in `commonCoreStrings`, `commonCoachStrings`, or `commonLearnStrings`
- EXCEPT for the [difficult question relative time string](https://www.figma.com/file/TfaSTT9gs6fPKhGdNUjffU/Channel-quizzes?node-id=1013%3A11166) `Attempted {{X time-unit}} ago` will need to be added to this component
- Routing and navigation has already been created for this user story

- In the main content area:
    - Add time for most recent attempt
    - Add last attempt information
    - Add the appropriate "answer annotations" string

### Criteria for success:
**Please copy these into your PR for "Steps for Testing"!**
_This issue is done when the Gherkin story below works_

- [ ] coach sees information about learners on the specified difficult question
```
Feature: Difficult question reports for independent practice quizzes from lessons
Scenario: coach sees detailed information for difficult questions
		Given that I am in 'Reports'
			And I click on the 'Lessons' subtab
			And I click on a lesson title
			And I click on the title of an independent practice quiz in the 'Reports' subtab
			And I click on the 'Difficult questions' subtab
		When I select the difficult question
		Then I see a list of learners who got the question incorrect, a preview of the question, and the option to show the correct answer.
```

**Comment by bjester** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Kolibri-Ecosystem-6047e525b8b9d900182e2d6f/issues/learningequality/kolibri/8517?planning-poker) with ZenHub @jredrejo @marcellamaki @sairina @rtibbles @MisRob @nucleogenesis @jamalex

**Comment by rtibbles** [comment]
May be able to leverage either existing Quiz or Exercise Difficult Question endpoints - main difference between the two is that Quiz counts skipped questions as difficult, whereas if a user has not engaged with an exercise question, that fact is not taken into account.

**Comment by rtibbles** [comment]
Fixed in #8521
